themes/bootstrap3/less/bootstrap.less

```diff
@@ -375,7 +375,13 @@ footer {
 }
 
 .dropdown-menu > li > .prompt-text {
-  &:extend(.dropdown-menu > li > a);
+  /* #LESS> */
+  &:extend(.dropdown-menu-ul-links);
+  /* <#LESS */
+  /* #SCSS>
+  @include dropdown-menu-ul-links;
```

Comment:
Why do we need SCSS-specific include instead of using `&:extend` for both? We already use the latter in quite a few places.